docs/getting-started/installation.md

```diff
@@ -5,11 +5,17 @@
 For virtualization and easy access, JanusGraph provides a [Docker image](https://hub.docker.com/r/janusgraph/janusgraph).
 Docker makes it easier to run servers and clients on a single machine without dealing with multiple installations.
 For instructions on installing and using Docker, please refer to the [docker guide](https://docker.com/get-started).
-Let's try running a simple JanusGraph instance in Docker:
+With JanusGraph server running in one container, we can either run the client in another container, or on our bare-metal machine.
+
+### Running Client inside another Docker container
```

Comment:
I think _Client_ isn't very specific as this could be any client that can access JanusGraph Server. So, I would say "the Gremlin Console" instead and use the same in other places in this article.